spec/index.bs

```diff
@@ -3885,17 +3891,23 @@ The following validation rules apply:
   - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
     {{GPUTextureUsage/COPY_DST}}.
 
-  For the texture {{GPUTexture/[[sampleCount]]}}:
-  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} must be equal to |destination|.
-    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
-  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is greater than 1:
-    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
-
-  For the texture {{GPUTexture/[[format]]}}:
-  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be equal to |destination|.
-    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
-  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
-    - The copy with |source|, |destination| and |copySize| must be a |copy of the whole subresource|.
+  For the |sampleCount|, |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}:
+  - |sampleCount| must be equal to
+    |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
+  - If |sampleCount| is greater than 1:
+    - |copiesWholeSubresources| must be true.
+
+  For the |format|, |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}:
+  - |format| must be equal to
+    |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+  - If |format| is a depth-stencil format:
+    - |copiesWholeSubresources| must be true.
+    - |source| and |destination| must refer to the same set of aspects
+        (depth, stencil, or both depth and stencil).
```

Comment:
On a related note, if we allow multi-aspect T2T copies then it makes about as much sense to allow multi-miplevel T2T copies, but honestly both are very unimportant. We could remove multi-aspect T2T copies.